stdlib/3/multiprocessing/__init__.pyi

```diff
@@ -79,6 +88,23 @@ def log_to_stderr(level: Optional[Union[str, int]] = ...) -> Logger: ...
 def Manager() -> SyncManager: ...
 def set_forkserver_preload(module_names: List[str]) -> None: ...
 def get_all_start_methods() -> List[str]: ...
-def get_context(method: Optional[str] = ...) -> BaseContext: ...
 def get_start_method(allow_none: bool = ...) -> Optional[str]: ...
 def set_start_method(method: str, force: Optional[bool] = ...) -> None: ...
+
+
+if sys.platform != "win32":
+    @overload
+    def get_context(method: None = ...) -> DefaultContext: ...
+    @overload
+    def get_context(method: Literal["spawn"]) -> SpawnContext: ...
+    @overload
+    def get_context(method: Literal["fork"]) -> ForkContext: ...
+    @overload
+    def get_context(method: Literal["forkserver"]) -> ForkServerContext: ...
+else:
+    @overload
+    def get_context(method: None = ...) -> DefaultContext: ...
+    @overload
+    def get_context(method: Literal["spawn"]) -> SpawnContext: ...
+    @overload
+    def get_context(method: str) -> BaseContext: ...
```

Comment:
Why did you include this overload for win32, but not for the other branch? I think we need it for both branches in case the method is not known at your check time.

Reply:
Yes you're right -- copy-paste failure here.

Reply:
Done!